index.php

```diff
@@ -294,7 +294,7 @@
                             <div class="slingshot-grid">
                             </div>
                             <div id="slingshot-pager">
-								<div class="switcher-checked" data-l10n-off><?php include('images/bullet.svg'); ?></div>
```

Comment:
I was actually having trouble properly aligning them as I couldn't tell how many tabs was used. VIM was also behaving strangely with the auto tabbing so I just put spaces as I thought it would be agreeable. Thank you for the correction.